examples/wordpress/wordpress-resources.yaml

```diff
@@ -0,0 +1,11 @@
+resources:
```

Comment:
Please add a note here that this is a Google Cloud Deployment Manager configuration